pkgs/games/corsix-th/default.nix

```diff
@@ -0,0 +1,17 @@
+{ fetchFromGitHub, lib, newScope, audioSupport ? true, movieSupport ? true, freetypeSupport ? true, buildDocs ? false, enableUnitTests ? false, buildAnimView ? true, buildLevelEdit ? true }:
+
+let
+    baseName = "corsix-th";
+    version = "0.64";
+    githubSource = fetchFromGitHub {
+        owner = "CorsixTH";
+        repo = "CorsixTH";
+        rev = "8741d41eb5de865c633e023bcb512b779c131c5e";
+        sha256 = "0chh9cv2kdc39sr0x8hclcyzd8dz2y6grgagqzkvr7j570wa5cqh";
```

Comment:
```suggestion
      owner = "CorsixTH";
      repo = "CorsixTH";
      rev = "v%{version}";
      sha256 = "0chh9cv2kdc39sr0x8hclcyzd8dz2y6grgagqzkvr7j570wa5cqh";
```

Reply:
This will have to be kept commit specific until 0.65 (which should be released soon!)